styleguide/package.json

```diff
@@ -1,45 +1,44 @@
 {
   "name": "ama-style-guide",
-  "version": "1.13.5",
+  "version": "3.5.0",
   "description": "A living style guide for the American Medical Association",
   "repository": "https://github.com/AmericanMedicalAssociation/AMA-style-guide",
   "main": "gulpfile.js",
   "dependencies": {
     "breakpoint-sass": "^2.7.1",
-    "browser-sync": "^2.19.0",
-    "gulp": "^3.9.1",
-    "gulp-autoprefixer": "^4.0.0",
-    "gulp-bump": "^2.9.0",
-    "gulp-clean": "^0.3.2",
-    "gulp-concat": "^2.6.1",
-    "gulp-cssmin": "^0.2.0",
-    "gulp-filter": "^5.0.1",
-    "gulp-gh-pages": "^0.5.4",
-    "gulp-git": "^2.4.2",
-    "gulp-if": "^2.0.2",
-    "gulp-imagemin": "^4.0.0",
-    "gulp-lintspaces": "^0.5.0",
-    "gulp-postcss": "^7.0.0",
-    "gulp-rename": "^1.2.2",
+    "browser-sync": "^2.23.5",
+    "gift": "^0.10.2",
+    "gulp-autoprefixer": "^4.1.0",
+    "gulp-bump": "^3.0.0",
+    "gulp-filter": "^5.1.0",
+    "gulp-gh-pages-gift": "^0.5.5",
+    "gulp-git": "^2.5.1",
+    "gulp-git-push": "^1.0.1",
+    "gulp-minify-css": "^1.2.4",
     "gulp-sass": "^3.1.0",
-    "gulp-shell": "^0.6.3",
-    "gulp-sourcemaps": "^2.6.1",
-    "gulp-stylelint": "^5.0.0",
-    "gulp-tag-version": "^1.3.0",
-    "gulp-uglify": "^3.0.0",
-    "gulp-util": "^3.0.8",
-    "p-wait-for": "^1.0.0",
+    "gulp-shell": "^0.6.5",
+    "gulp-tag-version": "^1.3.1",
     "patch-package": "^3.6.1",
     "postcss-reporter": "^5.0.0",
     "postcss-scss": "^1.0.2",
     "sass-grid": "^3.0.0"
   },
   "devDependencies": {
+    "argv": "0.0.2",
+    "del": "^3.0.0",
     "glob": "^7.1.2",
+    "gulp": "github:gulpjs/gulp#4.0",
+    "gulp-clean-css": "^3.9.2",
+    "gulp-concat": "^2.6.1",
+    "gulp-if": "^2.0.2",
     "gulp-plumber": "^1.1.0",
+    "gulp-rename": "^1.2.2",
     "gulp-sass-glob": "^1.0.8",
-    "gulp-svgmin": "^1.2.4",
-    "run-sequence": "^2.2.0"
+    "gulp-stylelint": "^6.0.0",
```

Comment:
If in the future we want to have travis run a linter (we we probably will) this needs to be a dependency not a dev dependency.